modin/config/envvars.py

```diff
@@ -627,6 +627,11 @@ class TestReadFromPostgres(EnvironmentVariable, type=bool):
     varname = "MODIN_TEST_READ_FROM_POSTGRES"
     default = False
 
+class ExperimentalNumPyAPI(EnvironmentVariable, type=bool):
+    """Set to true to use Modin's experimental NumPy API."""
+
+    varname = "MODIN_EXPERIMENTAL_NUMPY_API"
```

Comment:
Requires changes in https://modin.readthedocs.io/en/stable/flow/modin/config.html?